django_project/ta_system/models.py

```diff
@@ -1,17 +1,31 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    eagle_id = models.PositiveSmallIntegerField(null=True, blank=True)
+
+    def __str__(self):
+        return f'{self.user.username} Profile'
 
 
 class Course(models.Model):
-    course_num = models.CharField(max_length=14, unique=True, verbose_name="Course Number (e.g. CSCI110101)")
+    course_num = models.CharField(
+        max_length=14, unique=True, verbose_name="Course Number (e.g. CSCI110101)")
```

Comment:
Let's leave this on one line for now, as it was before.

Reply:
once I change all these to one line and save them, they automatically become two lines